helm/fluent-logshipping-app/templates/daemonset.yaml

```
@@ -98,13 +97,24 @@ spec:
       - name: fluent-bit-config
         configMap:
           name: {{ include "resource.default.name" . }}-configmap
+      ## Needed for syslog to work as the syslog plugins needs this id on AWS
       - name: etc-machine-id
         hostPath:
           path: /etc/machine-id
           type: File
-      - name: tmp-storage
+      - name: storage
+      {{- if .Values.fluentbit.persistence.pvc.enabled }}
+        persistentVolumeClaim:
+          {{- if .Values.fluentbit.persistence.pvc.existingClaim }}
+          claimName: {{ .Values.persistence.pvc.existingClaim }}
```

```suggestion
          claimName: {{ .Values.fluentbit.persistence.pvc.existingClaim }}
```
I think you forgot this

---

Thank you <3